impeller/entity/entity.cc

```diff
@@ -67,12 +70,36 @@ void Entity::IncrementStencilDepth(uint32_t increment) {
 
 void Entity::SetBlendMode(BlendMode blend_mode) {
   blend_mode_ = blend_mode;
+  if (BlendModeShouldCoverWholeScreen(blend_mode)) {
+    cover_whole_screen_ = true;
+  }
```

Comment:
If the blend mode gets set twice for some reason, this could get into an incorrect state. Maybe just `cover_whole_screen_ = BlendModeShouldCoverWholeScreen(blend_mode)`?

Reply:
Done.